src/libs/events.js

```diff
@@ -31,6 +31,7 @@ const eventsList = {
   catalogToggle: 'catalog:toggle',
   catalogView: 'catalog:view',
   catalogWorkspaceLink: 'catalog:workspaceLink',
+  catalogWorkspaceLinkExportFinished: 'catalog:workspaceLink:completed',
```

Comment:
Could you make sure this item gets added to our [mixpanel lexicon](https://mixpanel.com/project/2120588/view/19059/app/lexicon#transformations)?

Reply:
Yes, I will add it once this goes to prod, because I believe it will make the base event in the lexicon, and then I just need to edit the event.